package.json

```diff
@@ -50,5 +52,9 @@
   "packageManager": "pnpm@7.9.5",
   "optionalDependencies": {
     "svelte": "^3.54.0"
+  },
+  "dependencies": {
+    "ps-list": "^8.1.1",
```

Comment:
Do we really need this as deps, nor peerDeps?

Reply:
It is used for debug only and will be removed, once reason is found